packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +67,31 @@ export const Default = {
-
-        .long-content::-webkit-scrollbar-thumb {
-          background-color: transparent; /* make scrollbar transparent */
-          border-radius: 4px;
-        }
-
-        .long-content:hover::-webkit-scrollbar-thumb {
-          background-color: rgba(81, 81, 81, 0.9);
-        }
-      </style>
-      ${generateTemplate({
-        axis: {
-          x: {
-            type: 'slot',
-            name: 'content',
-            values: [
-              {
-                value: `<div class='long-content' slot='content' style='pointer-events:auto; width: 100px;'>Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliqu et nunc, quis ali qu am nisl nisl quis nisl.</div>`,
-                title: 'long with fixed width'
-              }
-            ]
-          }
-        },
-        args: overrideArgs([
-          {
-            type: 'attribute',
-            name: 'open',
-            value: true
-          },
-          {
-            type: 'attribute',
-            name: 'placement',
-            value: 'bottom'
-          }
-        ]),
-        constants: [
-          {
-            type: 'template',
-            name: 'width',
-            value: '<div class="template-position">%TEMPLATE%</div>'
-          }
-        ]
-      })}
-    </div>`;
-  }
-};
-
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['default', 'content'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'slot',
-          name: 'default',
-          title: 'slot=...',
-          values: [
-            {
-              value: `<div class="slot slot--border slot--background slot--text" style="--slot-content: ''; --slot-height: 24px; --slot-width: 24px;"></div>`,
-              title: 'default'
-            },
-            {
-              value: `<div class="slot slot--border slot--background slot--text" slot='content' style="--slot-content: ''; --slot-height: 22px; --slot-width: 82px;"></div>`,
-              title: 'content'
-            }
-          ]
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'width',
-          value: '<div class="template-position">%TEMPLATE%</div>'
-        },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    });
-  }
+export const Placement = {
+  name: 'Placement',
+  render: () => html` <sd-tooltip content="Lorem ipsum" placement="top" size="lg" trigger="click focus"></sd-tooltip> `
```

Comment:
This only shows the one placement (top) in the story. I would assume all the options would be either in the description, shown here in a flex-box or both.

Reply:
Placing them all in the story I think would be too much in my opinion, but agree that they can be added in the description.

Reply:
@Vahid1919 added the text with the options and included a couple of new stories, please review again